src/test/java/org/springframework/data/tarantool/TestUtils.java

```diff
@@ -0,0 +1,37 @@
+package org.springframework.data.tarantool;
+
+import lombok.SneakyThrows;
+
+import java.util.concurrent.Callable;
+
+public class TestUtils {
```

Comment:
Where is it used?

FYI there is a retry mechanism in the driver, you may find it helpful.